homeassistant/components/zwave_js/__init__.py

```diff
@@ -446,6 +447,14 @@ def async_on_notification(event: dict[str, Any]) -> None:
                     ATTR_ACKNOWLEDGED_FRAMES: notification.acknowledged_frames,
                 }
             )
+        elif isinstance(notification, MultilevelSwitchNotification):
+            event_data.update(
+                {
+                    ATTR_COMMAND_CLASS_NAME: "Multilevel Switch",
+                    ATTR_TYPE: notification.event_type,
+                    ATTR_PARAMETERS: notification.direction,
```

Comment:
based on the key this should be a list

Reply:
actually instead of parameters this should just be `direction` for the key

Reply:
as in ATTR_DIRECTION?